### ISSUE TITLE
noise/plopping-Sound at the very beginning of a audio track

### ISSUE THREAD

**Issue by oneoogeoo** [tpunix/SAROO#199]
In many games there is a noise/plopping-Sound at the very beginning of a audio track.
E.g. in Salamander Deluxe Pack with the Konami jingle.

**Comment by luixyto** [comment]
The same thing happens to me, there is an annoying POP when it loads. Depending on the game there are more or less POPS, I have tried it in 3 different saroos and in all 3 it does the same. Tested on a va0 jap

**Comment by luixyto** [comment]
any news of this bug?

**Comment by ragingchaos** [comment]
> any news of this bug?

I no longer get this bug with 0.7.

**Comment by kingchaz1973** [comment]
Hi Raging.

Does your copu play salamander 2 ? I get black screen on game load ?

**Comment by kingchaz1973** [comment]
> > any news of this bug?
> 
> 
> 
> I no longer get this bug in Salamander with 0.7. Still get pops in other games.

Hi does salamander 2 works.  Mines gives me black screen

**Comment by ragingchaos** [comment]
> Hi does salamander 2 works? Mines gives me black screen

Yes, check if you have a sector_delay in your config for that game.

**Comment by kingchaz1973** [comment]
> > Hi does salamander 2 works? Mines gives me black screen
> 
> Yes, check if you have a sector_delay in your config for that game.

Hi thanks for reply.  Iv added it and it works now.  Cheers

**Comment by oneoogeoo** [comment]
> > > any news of this bug?
> > 
> > 
> > I no longer get this bug in Salamander with 0.7. Still get pops in other games.
> 
> Hi does salamander 2 works. Mines gives me black screen

Your problem has nothing to do with the error reported here. 
Please do not post any off-topic questions in existing tickets!

**Comment by kingchaz1973** [comment]
> > > > any news of this bug?
> > > 
> > > 
> > > I no longer get this bug in Salamander with 0.7. Still get pops in other games.
> > 
> > Hi does salamander 2 works. Mines gives me black screen
> 
> Your problem has nothing to do with the error reported here. 
> Please do not post any off-topic questions in existing tickets!

Hi no problem.
